gdal/src/main/scala/geotrellis/contrib/vlm/gdal/GDALDataPath.scala

```diff
@@ -43,9 +45,24 @@ case class GDALDataPath(
   path: String,
   compressedFileDelimiter: String = "!"
```

Comment:
Is it ever the case that the compressed file delimiter changes or is it basically GDAL convention ?

Reply:
I'm not sure if it's a GDAL convention or not, but I believe it is in Rasterio. The only time I can think of would be if that character is used in a file name somewhere.